helm/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2024 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+ ## Current Developer Toolbox doesn't support environment files, make sure to remove any files or environment variables starting with $
+services:
+  platform:
+    image: benchmark:platform
+    container_name: platform
+    privileged: true
+    volumes:
+      - $log_dir:/tmp/results
+  igt:
+    image: benchmark:igt
+    container_name: igt
+    privileged: true
+    volumes:
+      - $log_dir:/tmp/results
+  # xpum:
+  #   image: benchmark:xpum
+  #   container_name: xpum
+  #   user: root
+  #   privileged: true
+  #   volumes:
+  #     - $log_dir:/tmp/xpumdump/
+  #     - /sys/firmware/acpi/tables/MCFG:/pcm/sys/firmware/acpi/tables/MCFG:ro 
+  #     - /proc/bus/pci/:/pcm/proc/bus/pci/ 
+  #     - /proc/sys/kernel/nmi_watchdog:/pcm/proc/sys/kernel/nmi_watchdog 
+  #   environment:
+  #     - XPUM_REST_NO_TLS=1
+  #   devices:
+  #     - /dev/dri:/dev/dri
+  #     - /dev/cpu:/dev/cpu
```

Comment:
remove the dead code?

Reply:
The XPU manager code is used for XEON systems. For the initial helm we can keep it there and add some documentation around uncommenting when running on XEON.

Reply:
good to know, it might worth to add your comment into the code so that new developers can learn this knowledge right there.